pkg/client/core/api/direct/github_direct.go

```diff
@@ -10,16 +11,16 @@ import (
 )
 
 type githubAPI struct {
-	client       github.Githuber
-	parameterAPI client.ParameterAPI
+	client  github.Githuber
+	service api.ParameterService
```

Comment:
Neat if you'll keep the services called something less generic than 'service'. Improves readability

Reply:
Mm, have not decided what I want it to be yet. Keeping iit generic so far.

Reply:
Thinking of the feature / tool refactoring will be a next natural step soon :)

Reply:
I meant; instead of service, use `parameterService`. Reading the code in the feature service hurts my eyes if I don't know what kind of service we're calling

Reply:
Also rename `client` to `githubClient` then?

Reply:
I won't harass anyone to do it when there was no change to it in the current PR, but yes please :D

Reply:
Don't really agree in such a small context - it's easy to see what service is and changing it to "parameterService" makes it more messy in my opinion. Anyway, don't care too strongly about this, so will change it.

Reply:
Didn't want to spend much time on this now as I want better naming conventions and will continue to work on this a little bit, but you guys are kind of forcing my hand, and I spend more time defending my non-actions than just doing it 😂